donut/modules/directory_search/templates/search_results.html

```diff
@@ -1,44 +1,106 @@
 {% extends 'layout.html' %}
 {% block page %}
-	<div class='container theme-showcase jumbotron' style='float: none; padding-bottom: 25px'>
-		<button class='btn' onclick='history.back()'>
-			&lt; Back
-		</button>
-		<div>Your search returned {{ users|length or 'no' }} results</div>
-		{% if users %}
-			<table class='table table-striped table-condensed'>
-				<thead>
-					<tr>
-						<th>Name</th>
-						<th>Graduation Year</th>
-						{% if show_images %}
-							<th> Image </th>
-						{% endif %}
-					</tr>
-				</thead>
-				<tbody>
-					{% for user in users %}
-						<tr>
-							<td>
-								<a href='{{ url_for("directory_search.view_user", user_id=user["user_id"]) }}'>
-									{{ user['full_name'] }}
-								</a>
-							</td>
-							<td>{{ user['graduation_year'] or '' }}</td>
-							{% if show_images %}
-								<td>
-									{% if user['image'] %}
-										<img
-											src='{{ url_for("directory_search.get_image", user_id=user["user_id"]) }}'
-											style='max-width: 100px; max-height: 150px'
-										/>
-									{% endif %}
-								</td>
-							{% endif %}
-						</tr>
-					{% endfor %}
-				</tbody>
-			</table>
-		{% endif %}
-	</div>
+<div class='container theme-showcase jumbotron' style='float: none; padding-bottom: 25px'>
+	<a class='btn btn-info' href='{{ url_for("directory_search.directory_search") }}'>Back</a>
+	<div>Your search returned {{ total }} results</div>
+	{% if users %}
```

Comment:
Can you fix the indentation? It's kinda hard to read what HTML and Jinja2 directives are nested inside what others. If you want to use spaces instead of tabs, that's fine.